public/new-image-card.html

```diff
@@ -187,26 +190,26 @@
           this.set('uploading_', true);
           this.$.uploadProgress.classList.remove('hidden');
           this.set('id_', Math.round(Math.random() * Math.pow(10, 16)));
-          this.uploadImage_(this.contentFile_, 'content');
-          this.uploadImage_(this.styleFile_, 'style');
-        },
-        uploadImage_: function(file, purpose) {
+          this.set('contentUploadProgress_', 0);
+          this.set('styleUploadProgress_', 0);
+          _.each(this.contentFile_, function(file, i, arr){
```

Comment:
add a space between closing paren and opening curly brace
https://github.com/airbnb/javascript#whitespace--before-blocks

do this in all new code